.github/workflows/release_library.yml

```diff
@@ -223,14 +227,12 @@ jobs:
   check-dependencies:
     name: check-dependencies
     uses: ./.github/workflows/check-nodejs-dependencies.yml
-    # Note: check-dependencies is failing for pantry due to "TypeError: Invalid Version: undefined"
```

Comment:
Ran into a similar issue while updating the custom action for monorepo support, so feeling good that we have this fixed now. If not, I added an input so that we can have pantry or any other library opt out of the dependency reporting manually (instead of hardcoding a repo name here)